invenio_base/app.py

```diff
@@ -181,10 +181,19 @@ def _loader(init_func, entry_points=None, modules=None):
     if entry_points:
         for entry_point in entry_points:
             for ep in pkg_resources.iter_entry_points(entry_point):
-                init_func(ep.load())
+                try:
+                    init_func(ep.load())
+                except Exception:
+                    app.logger.error(
```

Comment:
Exception is re-raised, so some logger higher up the hierarchy will catch it and cause double exception to be displayed if I used app.logger.exception.

The problem above code solves is if e.g. entry point loading fails, then you wouldn't see which modules was causing the problem and thus be blind as to where the problem was.